src/main/java/service/impl/InputHandler.java

```diff
@@ -0,0 +1,32 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import service.InputFileHandler;
+
+public class InputHandler implements InputFileHandler {
```

Comment:
Follow the encapsulation principle. Make your class and its members package-private if there is no need to expose them.